src/cmdline.rs

```diff
@@ -126,6 +126,7 @@ pub fn parse_args() -> Result<Config> {
                     Arg::with_name("image-url")
                         .short("u")
                         .long("image-url")
+                        .conflicts_with("image-file")
```

Comment:
We removed the default stream argument in #187 since the default is now offline installs. Hmm, I think that means we can now make all three conflict with each other. Let me try that out and update this comment if that works well.